src/components/application_manager/test/hmi_capabilities_test.cc

```diff
@@ -286,7 +286,6 @@ TEST_F(
 
   // Count of buttons in json file
   const size_t btn_length = buttons_capabilities_so.length();
-  EXPECT_EQ(16ull, btn_length);
   for (size_t index = 0; index < btn_length; ++index) {
```

Comment:
```suggestion
  EXPECT_EQ(17ull, btn_length);
  for (size_t index = 0; index < btn_length; ++index) {
```

Reply:
Discussed offline. This comment should not be considered